ci/kokoro/windows/build-dependencies.ps1

```diff
@@ -95,14 +95,32 @@ Get-Date -Format o
 $packages = @("zlib:x64-windows-static", "openssl:x64-windows-static",
               "protobuf:x64-windows-static", "c-ares:x64-windows-static",
               "grpc:x64-windows-static", "curl:x64-windows-static",
-              "gtest:x64-windows-static", "crc32c:x64-windows-static")
+              "crc32c:x64-windows-static")
 foreach ($pkg in $packages) {
     .\vcpkg.exe install $pkg
     if ($LastExitCode) {
         throw "vcpkg install $pkg failed with exit code $LastExitCode"
     }
 }
 
+# The dependencies are cached, we need to remove this old dependency. Otherwise
+# CMake files the old version of gtest+gmock instead of the external project.
+# This is clearly a defect in our hand-crafted caching, but without caching the
+# build takes about 1 hour to just build the dependencies.
```

Comment:
Isn't this a one-time operation? Once the cache is removed/redone these lines shouldn't be needed anymore.

Reply:
For a few hours (or days) builds based on the old version of `master` would recreate the cache.  Yes, we should remove this eventually though.